test/unittest/rtps/security/CMakeLists.txt

```diff
@@ -77,6 +77,7 @@ target_include_directories(SecurityAuthentication PRIVATE
     ${PROJECT_SOURCE_DIR}/test/mock/rtps/StatelessReader
     ${PROJECT_SOURCE_DIR}/test/mock/rtps/StatefulReader
     ${PROJECT_SOURCE_DIR}/test/mock/rtps/ReaderHistory
+    ${PROJECT_SOURCE_DIR}/test/mock/rtps/PDP
     ${PROJECT_SOURCE_DIR}/test/mock/rtps/PDPSimple
```

Comment:
Perhaps we can remove the PDPSimple mock?

Reply:
I am not sure, the SecurityTests rely on  `#include "SecurityTests.hpp"` in which PDPSimple is needed and used in some tests. If needed, we could refactor the tests to use pdp()

Reply:
Didn't realize. No need to change them now then. If required, we'll do that on a separate PR.